typespec-tests/src/main/java/com/type/union/IntsOnlyAsyncClient.java

```diff
@@ -19,7 +19,7 @@
 import com.type.union.implementation.IntsOnliesImpl;
 import com.type.union.implementation.models.SendRequest3;
 import com.type.union.models.GetResponse6;
-import com.type.union.models.GetResponseProp2;
+import com.type.union.models.GetResponseProp3;
```

Comment:
any idea why this number changes? Did they change the test case?

Reply:
Strange, I checked cadl-ranch, seems no change from last 24hours. Checked with Chenjie, the anonymous model logic does not change as well. 
But CI does not have generation diff, which means CI and action generates the same.

Reply:
CI and action should definitely have same result. That is expected that they get same result.

I am just not sure whether this change is due to TCGC 0.42.0 to 0.42.2. (basically, the only change would be from non-TCGC model to TCGC model; after TCGC model, upgrade of TCGC should not affect the number, as long as no change to test case).

Reply:
We can ignore it, if no clue.